tests/features/publication.feature

```diff
@@ -4,20 +4,28 @@ Feature: Publication content creation
   As an editor
   I need to be able to create and see publication items
 
+  @javascript
```

Comment:
Should we have also Functional/FunctionalJavascript tests for these changes? @ademarco @ademarco

Reply:
Why?

Reply:
Yep. It makes sense